manager/manager.go

```diff
@@ -491,6 +491,10 @@ func (m *Manager) Run(parent context.Context) error {
 	healthServer.SetServingStatus("Raft", api.HealthCheckResponse_NOT_SERVING)
 	localHealthServer.SetServingStatus("ControlAPI", api.HealthCheckResponse_NOT_SERVING)
 
+	if err := m.watchServer.Start(ctx); err != nil {
```

Comment:
I like the suggestion, but I'm worried it would trigger the error here, which is not recoverable:

https://github.com/moby/moby/blob/8d703b98b5c403743bf17e22395e32a7271b8d3c/daemon/cluster/noderunner.go#L212-L215

I'd prefer to just fix the regression in this PR, and a future change that's coordinated with some added retry/backoff in moby/moby could delay starting the watch server.